README.md

```diff
@@ -118,6 +132,24 @@ Below is an example configuration for a Lambda function that is part of a step f
 }
 ```
 
+Using the lambda:invoke API:
+```
+"States":{
+  "state1":{
+     "Type":"Task",
+     "Resource":"arn:aws:states:::lambda:invoke.waitForTaskToken",
+     "Parameters":{
+        "FunctionName":"arn:aws:lambda:us-west-2:ACCOUNT:function:FUNCTION_NAME",
+        "Payload":{
+           "size.$": "$.product.details.size",
+           "_lumigo.$": "$.['_lumigo', 'null']"
+        },
+        "Qualifier":"prod-v1"
+     },
+     "End":true
+  }
+}
+```
```

Comment:
Didn't understand, this is an API call to AWS API to configure a lambda step function to be traced?

Reply:
Following our discussion - reverted this change.